tests/test-pads.stanza

```diff
@@ -0,0 +1,59 @@
+#use-added-syntax(jitx)
+defpackage pads :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/land-patterns
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+pcb-landpattern lp (n:Int) :
+  for i in 0 to n by 4 do :
```

Comment:
Can you change the 4 here to a 6 to reflect the number of pads created per loop? This design will break if `num-pins` on line 51 is at least 5.